### PR TITLE
Ccs platt scale

### PR DESCRIPTION
**Pull request by lauritowal**
adding platt scaling to CCS. ~~Depends on https://github.com/EleutherAI/elk/pull/242~~

### CODE REVIEW

File: elk/training/ccs_reporter.py [hunk 1]
```diff
@@ -231,7 +231,9 @@ def reset_parameters(self):
 
     def forward(self, x: Tensor) -> Tensor:
         """Return the credence assigned to the hidden state `x`."""
-        return self.probe(self.norm(x)).squeeze(-1)
+        # return self.probe(self.norm(x)).squeeze(-1)
```
Comment (norabelrose) [inline elk/training/ccs_reporter.py:234]: Just remove this comment and then LGTM

**Review by norabelrose — CHANGES_REQUESTED** [review]
almost

**Review by AlexTMallen — APPROVED** [review]

File: elk/training/ccs_reporter.py (hunk 1 above)
Comment (lauritowal) [reply elk/training/ccs_reporter.py]: done

**Review by AlexTMallen — APPROVED** [review]

**Review by norabelrose — APPROVED** [review]
LGTM

**Pull request merged**